src/ScriptCs.Hosting/ScriptServicesBuilder.cs

```diff
@@ -48,7 +49,7 @@ public ScriptServices Build()
 
             _scriptEngineType = (Type)Overrides[typeof(IScriptEngine)];
 
-            var initDirectoryCatalog = _scriptName != null || _repl;
+            var initDirectoryCatalog = _scriptName != null || _repl || _loadScriptPacks;
```

Comment:
I am not sure I buy this yet. If I do what you are saying it will break all hosters today that use script packs.